app/views/candidate_interface/application_form/show.html.erb

```diff
@@ -8,30 +8,32 @@
   </div>
 <% end %>
 
-
 <h1 class="govuk-heading-xl govuk-!-margin-bottom-2">
   <%= t('page_titles.application_form') %>
 </h1>
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
     <p class="govuk-hint govuk-!-margin-bottom-8 govuk-!-margin-top-2"><%= @application_form_presenter.updated_at %></p>
-    
-    <% if FeatureFlag.active?('apply_again') && @application_form.apply_2? && @previous_application_form.application_choices.rejected.present?  %>
+
+    <% if FeatureFlag.active?('apply_again') && @application_form.apply_again? && @previous_application_form.application_choices.rejected.present?  %>
       <%= render(CandidateInterface::RejectionReasonsComponent.new(application_form: @previous_application_form)) %>
     <% end %>
 
-    <h2 class="govuk-heading-m govuk-!-font-size-27 govuk-!-margin-bottom-2">
-      <%= t('page_titles.course_choices') %>
-    </h2>
-    <% unless @application_form_presenter.application_choices_added? %>
-      <p class="govuk-body"><%= t('application_form.courses.intro') %></p>
+    <% if FeatureFlag.active?('apply_again') && @application_form_presenter.apply_again? %>
```

Comment:
Thinking about this feature flag. Is there actually any point to checking it's active outside of the bit where a candidate can hit the button to start the Apply Again process? 

Once they've duped their application and entered the apply_again phase, if we were to disable the feature flag for whatever reason in the future, then I think it would be preferable for any existing applications in that state to continue seeing the apply_again UI. Not doing so leaves the system in an incorrect state where they can, eg - add three courses to an apply_again application.

Reply:
@malcolmbaig 👍

Reply:
cc: @stevehook @davidgisbey for visibility of the above